content/en/docs/components/pipelines/user-guides/core-functions/run-a-pipeline.md

```diff
@@ -1,83 +1,150 @@
+   <img src="/docs/images/pipelines/start-a-run.png" 
+        alt="Start a run screen"
+        class="mt-3 mb-3 border border-info rounded">
 
+## Run Pipeline - KFP SDK Client
 
-## 2. Run from the KFP SDK client
-You may also programatically submit pipeline runs from the KFP SDK client. The client supports two ways of submitting runs: from IR YAML or from a Python pipeline function. For either approach, start by instantiating a `Client` using the `host` URL of your KFP instance:
+You may also programmatically submit pipeline runs from the KFP SDK client. 
+The client supports two ways of submitting runs: _from IR YAML_ or _from a Python pipeline function_. 
+
+{{% alert title="Note" color="dark" %}}
+See the [Connect the SDK to the API](/docs/components/pipelines/user-guides/core-functions/connect-api/) guide for more information about creating a KFP client.
+{{% /alert %}}
+
+For either approach, start by instantiating a [`kfp.Client()`][kfp-client]:
 
 ```python
-from kfp.client import Client
-client = Client(host='<YOUR_HOST_URL>')
+import kfp
+
+# TIP: you may need to authenticate with the KFP instance
+kfp_client = kfp.Client()
 ```
 
-To submit IR YAML for execution use the `.create_run_from_pipeline_package` method:
+To submit __IR YAML__ for execution use the [`.create_run_from_pipeline_package()`][kfp-client-create_run_from_pipeline_package] method:
 
 ```python
-client.create_run_from_pipeline_package('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
+#from kfp import compiler, dsl
+#
+#@dsl.component
+#def add(a: float, b: float) -> float:
+#   return a + b
+#
+#@dsl.pipeline(name="Add two Numbers")
+#def add_pipeline(a: float, b: float):
+#   add_task = add(a=a, b=b)
+#
+#compiler.Compiler().compile(
+#    add_pipeline, 
+#    package_path="./add-pipeline.yaml"
+#)
+
+kfp_client.create_run_from_pipeline_package(
+    "./add-pipeline.yaml", 
+    arguments={
+        "a": 1,
+        "b": 2,
+    }
+)
 ```
 
-To submit a Python pipeline function for execution use the `.create_run_from_pipeline_func` convenience method, which wraps compilation and run submission into one method:
+To submit a python __pipeline function__ for execution use the [`.create_run_from_pipeline_func()`][kfp-client-create_run_from_pipeline_func] convenience method, which wraps compilation and run submission into one method:
 
 ```python
-client.create_run_from_pipeline_func(pipeline_func, arguments={'param': 'a', 'other_param': 2})
+#from kfp import dsl
+#
+#@dsl.component
+#def add(a: float, b: float) -> float:
+#    return a + b
+#
+#@dsl.pipeline(name="Add two Numbers")
+#def add_pipeline(a: float, b: float):
+#    add_task = add(a=a, b=b)
+
+kfp_client.create_run_from_pipeline_func(
+    add_pipeline,
+    arguments={
+        "a": 1,
+        "b": 2,
+    }
+)
 ```
 
-See the [KFP SDK Client reference documentation][kfp-sdk-api-ref-client] for a detailed description of the `Client` constructor and method parameters.
+## Run Pipeline - KFP CLI
```

Comment:
I think the first step should be how to connect KFP CLI to the host

Reply:
Let's create a follow up issue, because I am not trying to add significant content in this PR, only improve formatting and layout.

I also don't have that information readily available to write up.